.github/workflows/test.yml

```diff
@@ -6,74 +6,35 @@ on:
   pull_request:
 
 # Current configuration:
-# - django 2.2, python 3.6, wagtail 2.6, sqlite
-# - django 2.2, python 3.7, wagtail 2.8, postgres
-# - django 3.0, python 3.8, wagtail 2.10, sqlite
-# - django 3.1, python 3.9, wagtail 2.12, postgres
-# - django 3.2, python 3.9, wagtail 2.14, sqlite
-# - django 4.0, python 3.9, wagtail 2.16, postgresql
-# - django 4.0, python 3.9, wagtail 3.0, sqlite (allow failures)
-# - django 4.1, python 3.10, wagtail 4.0, postgresql
-# - django 4.1, python 3.10, wagtail 4.1, sqlite
-# - django 4.1, python 3.10, wagtail master, postgres (allow failures)
+# - django 3.2, python 3.8, wagtail 4.1, sqlite
+# - django 4.1, python 3.9, wagtail 4.2, postgresql
+# - django 4.2, python 3.10, wagtail 5.0, sqlite
+# - django 4.1, python 3.11, wagtail main, postgres (allow failures)
 jobs:
   test:
     runs-on: ubuntu-20.04
     continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         include:
-          - python: 3.6
-            django: "Django>=2.2,<2.3"
-            wagtail: "wagtail>=2.6,<2.7"
-            database: "sqlite3"
-            experimental: false
-          - python: 3.7
-            django: "Django>=2.2,<2.3"
-            wagtail: "wagtail>=2.8,<2.9"
-            psycopg: "psycopg2==2.8.6"
-            database: "postgresql"
-            experimental: false
-          - python: 3.8
-            django: "Django>=3.0,<3.1"
-            wagtail: "wagtail>=2.10,<2.11"
-            database: "sqlite3"
-            experimental: false
-          - python: 3.9
-            django: "Django>=3.1,<3.2"
-            wagtail: "wagtail>=2.12,<2.13"
-            database: "postgresql"
-            psycopg: "psycopg2>=2.6"
-            experimental: false
-          - python: 3.9
+          - python: "3.8"
             django: "Django>=3.2,<3.3"
-            wagtail: "wagtail>=2.14,<2.15"
-            database: "sqlite3"
-            experimental: false
-          - python: 3.9
-            django: "Django>=4.0,<4.1"
-            wagtail: "wagtail>=2.16,<2.17"
-            database: "postgresql"
-            psycopg: "psycopg2>=2.6"
-            experimental: false
-          - python: 3.9
-            django: "Django>=4.0,<4.1"
-            wagtail: "wagtail>=3.0,<3.1"
+            wagtail: "wagtail>=4.1,<4.2"
             database: "sqlite3"
             experimental: false
-          - python: "3.10"
+          - python: "3.9"
             django: "Django>=4.1,<4.2"
-            wagtail: "wagtail>=4.0,<4.1"
+            wagtail: "wagtail>=4.2,<4.3"
             database: "postgresql"
             psycopg: "psycopg2>=2.6"
             experimental: false
           - python: "3.10"
-            django: "Django>=4.1,<4.2"
-            wagtail: "wagtail>=4.1,<4.2"
+            django: "Django>=4.2,<4.3"
+            wagtail: "wagtail>=5.0,<5.1"
```

Comment:
Nice one.

This is missing 5.1 (`wagtail>=5.1rc1,<5.2`) from the test matrix as far as I can see?

Reply:
Good idea! Added in ebceac7830478e668df0db115c618cdc01fe0ef3.